### PR TITLE
Helm Charts

### PR DESCRIPTION
**Pull request by netflash**
This is still WIP, in terms of lacking of README and so forth.
Anyway would like to get early feedback if possible.

Please keep in mind, that this is a public repo.

### CODE REVIEW

File: seal/values.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: quay.io/strataconsulting/seal:helm
```
Comment (dwin1000) [inline seal/values.yaml:4]: Is this quay repo a public repo to align with this GH repo being public?

Comment (netflash) [reply seal/values.yaml:4]: yes, it is

**Comment by toddmichael** [comment]
@netflash not sure if intentional, but your PR is pulling in commits, including merge commits, from others. Is this a rebase issue, perhaps?

**Comment by netflash** [comment]
There's something wrong with it. Thanks for noticing. 
Gonna recreate the PR to get rid of that.

**Pull request closed without merging**